docs/BOOTING.md

```diff
@@ -266,6 +266,16 @@ You will see a set of files in the current directory to locate into you tftp ser
 dhcp server to `ipxe.efi` (actually, it will use configuration from `ipxe.efi.cfg`). Files `kernel`, `initrd.img` and `initrd.bits`
 should be available via HTTP/HTTPs and you need to modify `ipxe.efi.cfg` with location of those files.
 
+## Boot options effect on TPM measurements (PCR-1)
+
+During the boot process, as stated by the TCG specification, BIOS/UEFI should measure the enumerated boot options into the TPM.
+UEFI measures the list of boot options and their configuration data in PCR-1. EVE is using PCR-1 as one of the sealing
+PCRs to protect the vault key from unauthorized access (check [Encrypted Data Store](security.md) for more details),
```

Comment:
This link gives a 404 since it needs to be upper case SECURITY. Might make sense to refer to the explicit section i.e., ./SECURITY.md#encrypted-data-store

Reply:
fixed.